client/gutenberg/extensions/presets/jetpack/utils/jetpack-logo.jsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import { Path, Polygon, SVG } from '@wordpress/components';
+import classNames from 'classnames';
+
+const JetpackLogo = ( { size = 32, className } ) => (
```

Comment:
I don't think we need any props
```suggestion
const JetpackLogo = () => (
```